pkg/3scale/amp/operator/backend_options_provider.go

```diff
@@ -216,6 +228,12 @@ func (o *OperatorBackendOptionsProvider) cronPodTemplateLabels() map[string]stri
 		labels[k] = v
 	}
 
+	if o.apimanager.Spec.Backend.CronSpec.Labels != nil {
```

Comment:
this `if` is not needed

Reply:
Addressed in this [commit](https://github.com/3scale/3scale-operator/pull/835/commits/f903bf41523372ad2297594e4efac295a12c0665)